### ISSUE TITLE
Small fix for LaunchWindows.bat and LaunchUnix.sh

### ISSUE THREAD

**Issue by StudioEtrange** [ThibautSF/ParadoxosModManager#35]
Launch files are not in your repository. 
I cant do a pull request to custom them. 
Could you please add LaunchWindows.bat and LaunchUnix.sh into the source code ?

I have made simple modification to take care of some path problem in windows and linux in these files.

LaunchWindows.bat
```
set _CURRENT_FILE_DIR=%~dp0
set _CURRENT_FILE_DIR=%_CURRENT_FILE_DIR:~0,-1%

cd /D %_CURRENT_FILE_DIR%
java -jar %_CURRENT_FILE_DIR%\ParadoxosModManager.jar
```

```
#!/bin/bash

_CURRENT_FILE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
cd "$_CURRENT_FILE_DIR"
java -jar "$_CURRENT_FILE_DIR"/ParadoxosModManager.jar
```

**Comment by ThibautSF** [comment]
Files moved in ParadoxosModManager/build/dist/

Thanks @StudioEtrange !

**Comment by ThibautSF** [comment]
Implemented in 0.7.0 : 6d364d06e3fe17d08b85aeacd22d28e232cb752c
